pkg/scripts/tempesta_installer.sh

```diff
@@ -90,13 +87,21 @@ tfw_install_packages()
 	shift
 	files=("${@}")
 
-	echo "Downloading latest packages from github.com/$GITHUB_USER/$repo ..."
-	mkdir -p $DOWNLOAD_DIR/$repo
-
-	for file in "${files[@]}"
-	do
-		tfw_download $repo $file
-	done
+  case $DISTRO in
+	"debian-8")
+    echo "Downloading latest packages from github.com/$GITHUB_USER/$repo ..."
+    mkdir -p $DOWNLOAD_DIR/$repo
+    ;;
+	"ubuntu-20")
+    repo=""
+    echo "Downloading latest packages from github.com/$GITHUB_USER/$repo ..."
+    # echo "$DOWNLOAD_DIR/$repo"
```

Comment:
We use git, so we always can fetch old code versions, so we don't leave commented out pieces of code